documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -106,6 +109,10 @@ The override configuration differs according to the listener type.
 For example, you can override hosts for `route`, DNS names or IP addresses for `loadbalancer`, and node ports for `nodeport`.
 <9> Authoization specified as `simple`, which uses the `AclAuthorizer` Kafka plugin.
 <10> (Optional) Super users can access all brokers regardless of any access restrictions defined in ACLs.
++
+WARNING: An OpenShift Route address comprises the name of the Kafka cluster, the name of the listener, and the name of the namespace it is created in.
```

Comment:
Use "project" instead of "namespace" when referring to OpenShift.

Reply:
It's used here to make it clearer what it relates to in the config example: the `namespace` property